kolla-ansible/ansible/roles/triliovault/defaults/main.yml

```diff
@@ -109,6 +109,12 @@ triliovault_keystone_user: "dmapi"
 triliovault_keystone_password: "NSqfUIUseG9U23j7ae3ZjiVg43mrnlOll0osgrIc"
 
 
+triliovault_ks_users:
+  - project: "service"
+    user: "{{ triliovault_keystone_user }}"
+    password: "{{ triliovault_keystone_password }}"
```

Comment:
It good if we use similar and simple variable names

like 
triliovault_keystone_user -> triliovault_ks_user
triliovault_keystone_password -> triliovault_ks_password